src/app/components/user/cognito-user-edit.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from 'react';
+import {
+  arrayOf,
+  func,
+  string,
+} from 'prop-types';
+import {
+  Card,
+  CardHeader,
+  CardContent,
+  FormControlLabel,
+  Switch,
+  ThemeProvider,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+
+import theme from '../../themes/theme';
+import { ChplTextField } from '../util';
+import { ChplActionBar } from '../action-bar';
+import {
+  user as userPropType,
+} from '../../shared/prop-types';
+
+const useStyles = makeStyles(() => ({
```

Comment:
```suggestion
const useStyles = makeStyles({
```

Making it a function was necessary when we had to pass some sort of value into the makeStyles call. We don't, so having it a function is overkill